com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickPool.cs

```diff
@@ -354,13 +356,18 @@ static void SetPixel(DynamicArray<Color> data, int x, int y, int z, int dataLocW
             data[index] = value;
         }
 
-        public static void FillDataLocation(ref DataLocation loc, SphericalHarmonicsL2[] shl2, int startIndex, int count, ProbeVolumeSHBands bands)
+        public static void FillDataLocation(ref DataLocation loc, ProbeVolumeSHBands srcBands, NativeArray<Color> shData, int startIndex, int count, ProbeVolumeSHBands dstBands)
```

Comment:
Pass in source data bands and desired destination bands separately so we can correctly fill in missing data where applicable.

Reply:
I'd add a comment before the function to make sure that the reader of only this function knows that the shData is preswizzled in the format that the 3d texture expects.